cinema/views.py

```diff
@@ -56,3 +91,48 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        query_set = (
```

Comment:
you already defined queryset with annotations above - just access it through self.